internal/overlord/servstate/manager.go

```diff
@@ -410,11 +410,13 @@ func (m *ServiceManager) SendSignal(services []string, signal string) error {
 	for _, name := range services {
 		s := m.services[name]
 		if s == nil {
+			errors = append(errors, fmt.Sprintf("cannot send signal to %q: service is not running", name))
```

Comment:
Oh, hey, or maybe not? :-)  Maybe that default clause is the one poorly worded indeed?

Reply:
Fixed.